src/main/java/uk/gov/justice/laa/crime/cfecrime/cma/stubs/LocalCmaService.java

```diff
@@ -1,5 +1,6 @@
 package uk.gov.justice.laa.crime.cfecrime.cma.stubs;
 
+import lombok.Setter;
```

Comment:
We're trying to use immutable objects in this project after being bitten by side-effects in cfe-civil.